src/Microsoft.AspNet.Mvc.WebApiCompatShim/project.json

```diff
@@ -13,12 +13,16 @@
     "frameworks": {
         "aspnet50": {
             "frameworkAssemblies": {
-                "System.Net.Http": "4.0.0.0"
+                "System.Net.Http": "4.0.0.0",
+                "System.Xml": "",
```

Comment:
They are from GAC, so we need not add version numbers...i fixed the one with System.Net.Http too now to not specify version numbers..